src/site/content/en/blog/bfcache/index.md

```diff
@@ -303,7 +303,15 @@ onAllChangesSaved(() => {
 {% endCompare %}
 
 
-### Avoid window.opener references
+### Do not serve pages with `Cache-control: no-store`
+
+`Cache-control: no-store` is an HTTP header web servers can set on responses to ensure the response is not stored in any cache. When this is set on the page itself (as opposed to any subresource), this means that page will not be included the bfcache. There is [a proposal to change this](https://bugs.chromium.org/p/chromium/issues/detail?id=1228611) but at present any pages sending this are ineligible to use the bfcache.
+
+For sites that simply want to ensure an up-to-date page is used each time, using `Cache-Control: no-cache` or `Cache-Control: max-age=0` prevents the page being used without being revalidated but, unlike `Cache-control: no-store`, still allows the bfcache to be used. For bfcache navigations a page refresh may not be expected or desirable for the user.
```

Comment:
```suggestion
For sites that simply want to ensure an up-to-date page is used each time, using `Cache-Control: no-cache` or `Cache-Control: max-age=0` prevents the page from being used without revalidation first. And unlike `Cache-Control: no-store`, these directives do not disqualify a page from being eligible for bfcache.
```

I'm not sure that last sentence is necessary, as the rest of the post makes the argument why bfcache is a better user experience. I don't think you need to restate it here.

Reply:
I specifically added the last sentence to answer the question of "why should back/forward navigations be special?". For example, if I'm a news publisher I might prefer the latest page is used, but I argue on back/forward navigations (specifically on "back") a fresh page might not be expected rather than going back to the page you were just at.

WDYT? Still not worth including that last sentence?

The rest of your edits are find btw so will commit them with or without the last sentence as you see fit.